spec/javascripts/fixtures/NewsletterSticky.html

```diff
@@ -1,9 +1,11 @@
 <div id="fixture-1">
   <div class="news-signup-sticky news-signup-sticky--positioned" data-dough-component="NewsletterSticky">
     <form onsubmit="return false;">
-      <button class="news-signup-sticky__close">
+      <button class="news-signup-sticky__close" id="first-focusable">
```

Comment:
The `t-*` MUST NOT be removed or the automated test suites will break. Sorry to seem shouty here, but they are not part of a one-off test cycle.

They are vital hooks into the HTML/view layer for automated end to end tests that stop us having to depend on the structure of the DOM to find the things we need to check. This makes the test less brittle as changes to the visual design are less likely to break the tests.

Please don't remove these unless you can be sure there are no references to them anywhere under the `/spec` folder of a project.